king_phisher/plugins.py

```diff
@@ -607,6 +610,19 @@ def load_module(self, name, reload_module=False):
 			recursive_reload(module)
 		return module
 
+	def install_packages(self, packages):
```

Comment:
This method should be documented and be sure to remember to add the `.. versionadded::` specifier.